integrated_channels/integrated_channel/exporters/content_metadata.py

```diff
@@ -116,7 +116,12 @@ def export(self, **kwargs):
                         self.enterprise_configuration,
                         json.dumps(transformed, indent=4),
                     )
-                    content_last_modified = item.pop('content_last_modified')
+
+                    # There are some scenarios where `content_last_modified` isn't present in the fetched content
```

Comment:
Are we concerned at all about which scenarios these are?  Any reason to write a log entry here?

Reply:
logs added!